.github/workflows/test-upstream.yml

```diff
@@ -44,6 +45,12 @@ jobs:
         with:
           toolchain: stable
           override: true
+      - name: Update upstream cargo dependencies
+        run: |
+          cd dask_planner
+          mv test-upstream.toml Cargo.toml
+          cat Cargo.toml
+          cargo update
```

Comment:
Do we also want to do this as part of the option upstream testing for PRs (which is enabled with commits containing `[test-upstream]`)